sec.club/src/components/EventBanner/EventBanner.test.js

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import { render, unmountComponentAtNode } from "react-dom"
+import { mount } from 'enzyme'
+import "@testing-library/jest-dom"
+
+import EventBanner from "./EventBanner.jsx"
+
+let container = null;
+describe("Event container appearance", () => {
+    beforeEach(() => {
+        container = document.createElement("div")
+    })
+
+    afterEach(() => {
+        unmountComponentAtNode(container)
+    })
+
+    it("should render without crashing", () => {
+        render(<EventBanner flyerSource="LearnHack.png"/>, container)
```

Comment:
The LearnHack flyer may be renamed or removed in the future, which would break these tests.

Change the image used to test to a placeholder image meant only for use in these tests. Make it small in file size, but keep the same aspect ratio as the other flyers. Put placeholder text on the image, similar to the test article.